### PR TITLE
a few more test cases for lsh

### PR DESCRIPTION
**Pull request by arvidn**
demonstrating stripping leading zeros and adding a leading zero to ensure the result is unsigned

### CODE REVIEW

File: tests/brun/lsh-d.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+brun -c '(lsh (q . 0xffffff) (q . 0))'
+cost = 76
+0x00ffffff
```
Comment (aqk) [inline tests/brun/lsh-d.txt:3]: Ah, yes, that's a good one. I'll look and see if we have the same for ash

Comment (aqk) [reply tests/brun/lsh-d.txt:3]: I see this one

> edge-cases/ash-06.txt
> 1:brun '(ash (q . 0xffff) (q . 0xffff))'

**Review by aqk — APPROVED** [review]
